src/NServiceBus.Core/ObjectBuilder/LightInject/LightInject.g.cs

```diff
@@ -32,7 +32,7 @@
 #endif
 
 #if NETCOREAPP2_0
-#define NETSTANDARD13
```

Comment:
These defines were added by @timbussmann. I changed it because netstandard1.6 is a better match for netcoreapp2.0 than netstandard1.3.

For example, if I had a netcoreapp2.0 project add added a NuGet package to it that had both 1.3 and 1.6 assemblies in it, the 1.6 one would be picked.